app/services/processors/jira_project_development_cycle_updater.rb

```diff
@@ -43,5 +47,21 @@ def issue_update!(issue, issue_fields)
     def issue_type_field(issue_fields)
       issue_fields[:issuetype][:name]
     end
+
+    def in_progress_at(histories)
+      return unless histories
+
+      @in_progress_at = nil
+      histories.each do |history|
+        items = history[:items]&.first
```

Comment:
I'm a bit confused with this each-do loop with a single assignment to `@in_progress_at, could you explain me what is the idea?
Also, Are you using that variable?

I am thinking if maybe there is a reduced and simpler way to do it?

Reply:
The histories hash has all the updates of an issue. So that I need to find the history which change their status from 'To Do' to 'In Progress'. Maybe there is a better way but I couldn't find it.

The variable is returned and use in the issue update.

Reply:
And what about breaking if you find it or something like that?

Is the variable necessary?, Couldn't you just return the value? or else use @ in the  issue update method.

Sorry if I am not getting something. 🙈

Reply:
I've change it to use `find` method I think is a better approach and avoid variables.